modules/dreamview/backend/sim_control/sim_control.h

```diff
@@ -126,12 +130,18 @@ class SimControl : SimControlInterface {
   // Number of planning received in terms of one RoutingResponse.
   int planning_count_ = -1;
 
+  // Whether planning has requested a re-routing.
   bool re_routing_triggered_ = false;
 
-  // Whether the sim control is enabled / initialized.
+  // Whether the sim control is enabled.
   bool enabled_ = false;
+
+  // Whether the adapter setup has been initialized.
   bool inited_ = false;
```

Comment:
likewise, inited_ seems to be too broad, maybe rename it to something more explicit?